src/Microsoft.ML.Transforms/Text/StopWordsRemovingTransformer.cs

```diff
@@ -602,32 +668,31 @@ private static VersionInfo GetVersionInfo()
                 verReadableCur: 0x00010001,
                 verWeCanReadBack: 0x00010001,
                 loaderSignature: LoaderSignature,
-                loaderAssemblyName: typeof(CustomStopWordsRemovingTransformer).Assembly.FullName);
+                loaderAssemblyName: typeof(CustomStopWordsRemovingTransform).Assembly.FullName);
         }
 
-        public const string StopwordsManagerLoaderSignature = "CustomStopWordsManager";
-        private static VersionInfo GetStopwrodsManagerVersionInfo()
+        private const string StopwordsManagerLoaderSignature = "CustomStopWordsManager";
```

Comment:
>StopwordsManagerLoaderSignature [](start = 29, length = 31)

do you even want to define a variable for it.. the compiler will probably optimize it away.